dpnp/backend/tests/test_utils_iterator.cpp

```diff
@@ -429,6 +429,40 @@ TEST_P(IteratorReduction, sycl_reduce_axis)
     }
 }
 
+TEST(TestUtilsIterator, sycl_get_first)
```

Comment:
`sycl_get_first` not shure about this name.

Reply:
I'm not going to merge the test to master, because the test is overlapped by other existing test. It's just a simple issue reproducer.

Reply:
see https://github.com/IntelPython/dpnp/pull/661#discussion_r601578733